src/auth/tenant.ts

```diff
@@ -155,15 +192,30 @@ export class Tenant {
         allowPasswordSignup: false,
       });
     }
+    if (typeof response.mfaConfig !== 'undefined') {
```

Comment:
response is typed any in this constructor. Can it be changed to `TenantServerResponse`?

Reply:
Done